### ISSUE TITLE
How to indent to height of status bar in 8.0.0-b03

### ISSUE THREAD

**Issue by beeline09** [mikepenz/MaterialDrawer#2563]
![Screenshot_1582705020](https://user-images.githubusercontent.com/2519627/75325467-ef7e0a00-5889-11ea-8641-760f565f98ac.png)

As far as I remember, in the old versions it was called translucentStatusBar, but in the new version I do not find anything like it.

**Comment by mikepenz** [comment]
The new v8 will actually fully adjust to what the system provides. it looks like your theme / ui configuration makes the UI draw full height (behind the statusbar) but you have given the statusbar a solid color in addition. 

When using a `DrawerLayout` you should apply the scrim on the `DrawerLayout` and not on the system ui. 

Please note that this behavior is `DrawerLayout` behavior and system behavior. 

The inset for the header is automatically applied via the window insets there are. it seems the window insets in your case were consumed before though. 

Check if you have the fit system windows: 
https://github.com/mikepenz/MaterialDrawer/blob/develop/app/src/main/res/layout/activity_sample.xml#L6

**Comment by mikepenz** [comment]
https://github.com/mikepenz/MaterialDrawer/blob/develop/app/src/main/res/layout/activity_sample.xml#L48

**Comment by beeline09** [comment]
I use fitSystemWindows
`<?xml version="1.0" encoding="utf-8"?>
<layout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools">

    <data>

        <variable
            name="vm"
            type="pd.merch.ru.viewModels.MainActivityVM" />
    </data>


    <androidx.drawerlayout.widget.DrawerLayout
        android:id="@+id/root"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        android:fitsSystemWindows="true">

        <androidx.coordinatorlayout.widget.CoordinatorLayout
            android:id="@+id/clMain"
            android:layout_width="match_parent"
            android:layout_height="match_parent"
            android:background="@drawable/fon"
            android:gravity="center"
            tools:context=".MainActivity">

            <com.google.android.material.appbar.AppBarLayout
                android:id="@+id/appBarLayout"
                android:layout_width="match_parent"
                android:layout_height="wrap_content">

                <androidx.appcompat.widget.Toolbar
                    android:id="@+id/toolbar"
                    android:layout_width="match_parent"
                    android:layout_height="?attr/actionBarSize"
                    android:background="?attr/colorPrimary"
                    app:layout_scrollFlags="enterAlways"
                    app:popupTheme="@style/AppTheme.AppBarOverlay"
                    app:theme="@style/AppTheme.AppBarOverlay"
                    app:title="@{vm.toolbarTitle}"
                    app:subtitle="@{vm.toolbarSubTitle}"
                    tools:title="ТоргПред ПД"
                    tools:subTitle="Авторизация"/>


            </com.google.android.material.appbar.AppBarLayout>

            <androidx.constraintlayout.widget.ConstraintLayout
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                android:background="#eedddddd"
                app:layout_behavior="@string/appbar_scrolling_view_behavior">

                <fragment
                    android:id="@+id/mainFragment"
                    android:name="androidx.navigation.fragment.NavHostFragment"
                    android:layout_width="match_parent"
                    android:layout_height="match_parent"
                    app:defaultNavHost="true"
                    app:navGraph="@navigation/main_graph"
                    tools:context=".MainActivity"
                    tools:showIn="@layout/activity_main" />
            </androidx.constraintlayout.widget.ConstraintLayout>

        </androidx.coordinatorlayout.widget.CoordinatorLayout>

        <com.mikepenz.materialdrawer.widget.MaterialDrawerSliderView
            android:id="@+id/slider"
            android:layout_width="wrap_content"
            android:layout_height="match_parent"
            android:layout_gravity="start"
            android:fitsSystemWindows="true" />
    </androidx.drawerlayout.widget.DrawerLayout>



</layout>`

Here is my App theme:

    <style name="AppTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
        <item name="colorPrimary">@color/colorPrimary</item>
        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
        <item name="colorAccent">@color/colorAccent</item>
        <item name="windowNoTitle">true</item>
        <item name="windowActionBar">false</item>
        <item name="toolbarStyle">@style/ToolbarMy</item>
        <item name="materialDrawerStyle">@style/Widget.MaterialDrawerStyle</item>
        <item name="materialDrawerHeaderStyle">@style/Widget.MaterialDrawerHeaderStyle</item>
    </style>

**Comment by mikepenz** [comment]
If you want to have a translucent statusbar you want to tell this to the theme too
```xml
<item name="android:windowTranslucentStatus">true</item>
```

Do you apply some window flags? anything which may consume the insets?

![photo_2020-02-26 09 47 13](https://user-images.githubusercontent.com/1476232/75327599-fa7e6d80-587c-11ea-8d97-2fe241f4de18.jpeg)

**Comment by beeline09** [comment]
No, i don't apply some window flags. But after i apply in <item name="android:windowTranslucentStatus">true</item> in my theme i see this:
![Screenshot_1582707073](https://user-images.githubusercontent.com/2519627/75328009-59001780-588e-11ea-88d8-8b75918b8545.png)
but also no margin from top to statusbar height...

**Comment by mikepenz** [comment]
May you try and install the sample app and show how it behaves? 

What's the android version of your phone?

**Comment by beeline09** [comment]
I found a problem. I use a view instead of an account in this custom view I did not add `android:fitsSystemWindows="true"`
After adding this to my view, the indentation worked.
![Screenshot_1582708670](https://user-images.githubusercontent.com/2519627/75330374-09bbe600-5892-11ea-8374-89fc5cc2de0d.png)

I could not even think that this should be done for any enclosed view.

I use  android emulator With Pixel 3 XL frame and Android API R

Problem is solved. Thank you!
